src/Microsoft.Data.Analysis/PrimitiveDataFrameColumnComputations.tt

```diff
@@ -65,6 +65,8 @@ namespace Microsoft.Data.Analysis
     }
 
 <# foreach (TypeConfiguration type in typeConfiguration) { #>
+<# if (type.TypeName == "DateTime") {#>
```

Comment:
Is this current plan that no computation function will be supported for `DataTime`

Reply:
There is an existing file called DateTimeComputations that have all the date time computations. They aren't similar to the other primitive types so I just left it as is. I've added a comment here.